chroma_help/help.py

```diff
@@ -14,6 +14,9 @@
     "start_copytool": "Start this copytool to begin processing HSM requests.",
     "stop_copytool": "Stop this copytool. In-progress HSM actions will be canceled.",
     "configure_copytool": "Configure this copytool on the worker.",
+    "configure_stratagem": "Configure's Stratagem on the selected node.",
+    "configure_stratagem_long": "Configure's Stratagem on the selected node.",
+    "configure_stratagem_description": "Configure Stratagem description",
```

Comment:
Fixme

Reply:
Do we know what these values should be?

Reply:
Can you describe where you are using the values above?